### PR TITLE
added a user info on the dashboard

### PR DESCRIPTION
**Pull request by Ganem1227**

### CODE REVIEW

File: src/pages/userdata/UserAccountInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Container, Row, Col, Image } from 'react-bootstrap';
+
+const DisplayUserInfo = (user) => (
+	<Container className="bg-secondary shadow-1-strong rounded-2">
+        <Row>
+            <h1>Account Information</h1>
+        </Row>
+        <Row>
+            <Col lg="2">
+                <Image src="https://c4.wallpaperflare.com/wallpaper/392/1018/527/flcl-red-background-cat-animals-wallpaper-preview.jpg" rounded fluid height="auto"></Image>
```
Comment (jadeallencook) [inline src/pages/userdata/UserAccountInfo.tsx:10]: Swap out the source to point to the local.

File: src/pages/userdata/UserAccountInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Container, Row, Col, Image } from 'react-bootstrap';
+
+const DisplayUserInfo = (user) => (
```
Comment (jadeallencook) [inline src/pages/userdata/UserAccountInfo.tsx:3]: Deconstruct the user here:
`{ name, subjectPronoun }`

File: src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import { Container } from 'react-bootstrap';
+import UserAccountInfo from './userdata/UserAccountInfo';
```
Comment (jadeallencook) [inline src/pages/Dashboard.tsx:2]: This component should live in `components/organisms`.

File: package.json [hunk 4]
```diff
@@ -23,13 +23,16 @@
 		"@types/node": "^18.7.18",
 		"@types/react": "^18.0.20",
 		"@types/react-dom": "^18.0.6",
+		"@types/react-router-dom": "^5.3.3",
 		"bootstrap": "^5.2.2",
 		"firebase": "^9.10.0",
+		"firebase-admin": "^11.2.1",
+		"firebase-functions": "^4.0.2",
```
Comment (jadeallencook) [inline package.json:30]: What are these new packages?

**Review by jadeallencook — CHANGES_REQUESTED** [review]

File: src/pages/userdata/UserAccountInfo.tsx (hunk 2 above)
Comment (jadeallencook) [reply src/pages/userdata/UserAccountInfo.tsx:3]: Also, please use the `UserInferface` here.

File: package.json (hunk 4 above)
Comment (Ganem1227) [reply package.json:30]: i'm not sure. I installed the framework and it just added them. I can try removing them and see if something breaks.

Comment (Ganem1227) [reply package.json]: nothing broke

**Comment by jadeallencook** [comment]
This page was added in the dashboard update.

**Pull request closed without merging**